operator/internal/controller/konflux_controller.go

```diff
@@ -105,16 +98,6 @@ func (r *KonfluxReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 	log.Info("Reconciling Konflux", "name", konflux.Name)
 
-	// Apply all embedded manifests
-	if err := r.applyAllManifests(ctx, konflux); err != nil {
-		log.Error(err, "Failed to apply manifests")
-		SetFailedCondition(konflux, ConditionTypeReady, "ApplyFailed", err)
-		if updateErr := r.Status().Update(ctx, konflux); updateErr != nil {
-			log.Error(updateErr, "Failed to update status")
-		}
-		return ctrl.Result{}, err
-	}
-
 	// Apply the KonfluxApplicationAPI CR
 	if err := r.applyKonfluxApplicationAPI(ctx, konflux); err != nil {
```

Comment:
Don't we want to refactor this function to fit all the components in this PR as well?

Reply:
How do you mean? It follows the same pattern as the rest of the components did.

Reply:
It is, but we mentioned that when all the components will be onboarded, we will refactor this part.
It can be in another PR

Reply:
Refactor how?